activestorage/app/models/active_storage/variant.rb

```diff
@@ -55,6 +55,7 @@
 class ActiveStorage::Variant
   attr_reader :blob, :variation
   delegate :service, to: :blob
+  delegate :content_type, :filename, to: :specification
```

Comment:
@fleck this seems to duplicate the definition at line 128 of this same file as now I see these warnings:
```
activestorage/app/models/active_storage/variant.rb:128: warning: method redefined; discarding old filename
activestorage/app/models/active_storage/variant.rb:59: warning: previous definition of filename was here
activestorage/app/models/active_storage/variant.rb:128: warning: method redefined; discarding old content_type
activestorage/app/models/active_storage/variant.rb:59: warning: previous definition of content_type was here
```

Reply:
That’s me. I missed the definition at line 128. I’ll fix.

Reply:
Fixed in 57d926a78a.